src/test/kotlin/pt/up/fe/ni/website/backend/utils/documentation/payloadschemas/model/PayloadProject.kt

```diff
@@ -34,6 +34,12 @@ class PayloadProject : ModelDocumentation(
         DocumentedJSONField(
             "targetAudience",
             "Information about the target audience",
+            JsonFieldType.STRING,
+            optional = true
+        ),
+        DocumentedJSONField(
+            "github",
+            "Handle/link to the project's GitHub repository",
```

Comment:
Same question as above, if `optional` indicates it is not required, then we should add it here (and in slogan) too